src/guides/using-final-ik.md

```diff
@@ -0,0 +1,107 @@
+
+Much better! This looks good, but Final IK provides a handful of built-in components that we can use to polish this animation even further.
+
+## Using Final IK components
+
+Our character works well, but there are a few places where it falls short. Our character doesn't lean when running in a circle, and if we add a ramp to the scene that our character can walk up, you can see the character's feet don't line up with the ground.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+Luckily, Final IK comes with a variety of components that can be used to refine the animation of our character.
+
+On the "Pilot" game object, add a **FullBodyBipedIK** component, a **BodyTilt** component.
+
+Populate the **BodyTilt** component **IK** field by dragging in a reference to the **FullBodyBipedIK** component. We'll also need to provide a left and right pose. Normcore includes a "Pose Left" and "Pose Right" preset that can be found in the `Normal/Examples/Final IK Player Controller/Character` folder.
+
+You can adjust the values for **Tilt Speed**, and **Tilt Sensitivity** as you like, but we have found 5.0 for Tilt Speed and 0.2 for Tilt Sensitivity look great.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+Last, let's add the **GrounderFullBodyBiped** component. Open up the Solver, and set *Layers* to "Default". This is the collision layer used for the solver. As your game grows, you may want to create a dedicated layer for the ground and surfaces you would like your player to land on.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+Now that both of those components are set up. Let's test it out! Have the player run in a circle and walk up the ramp.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Ideally this video is a frame by frame replay of the first video where it didn't work, but now it works)
+
+Much better! Our character now runs around and looks great doing it, but our camera is left behind. Let's fix that!
+
+## Camera controls
+
+Normcore comes with a built-in component called **PlayerCameraManager**. This component handles all of the logic for moving the camera with your player. It also supplies metadata about where the camera is looking so the **Player** component can correctly handle things like strafing.
+
+Add the **PlayerCameraManager** component to the "Player" game object and create a child game object called "Follow Target". This will represent the point on the player that the camera should follow. Wire it up to the **Follow Target** slot and we're good to go. (TODO: Player should automatically find the **PlayerCameraManager** component if the're on the same game object)
+
```

Comment:
I had to move the camera target up by 1 to get it center better on the character.

Reply:
Fixed in [c8d696f](https://github.com/NormalVR/Normcore-Documentation/pull/16/commits/c8d696fa82d5caebaf36a23ae688e2ffafcaefc4).